COMPONENT_MATURITY.md

```diff
@@ -0,0 +1,170 @@
+# Component maturity
+
+Each component is tested against the following items to guage the component's maturity. Use this list to understand how to test the component and when to make a checklist item 'Not applicable'.
+
+## Accessibility
+### WCAG 2.1 Level AA Conformance
+All Axe and Storybook plugins for WCAG AA compliance have passed.
+#### About
+Code is checked with the [Axe Core plugin (external link)](https://github.com/dequelabs/axe-core "Axe core plugin repository") that finds on average 57% of WCAG issues automatically. You can see the complete list of rules that are checked in the [rules descriptions (external link)](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md "Axe core plugin rules description"). 
+
+You can use Storybook to test for WCAG violations and color blindness filters.
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+Run `yarn test:a11y`
+
+### Supports Forced Colors Mode (FCM)
+While using FCM the components text is legible and improves readability.
+
+#### About
```

Comment:
Looking at the 'About' sections, I think most of the about sections are related to "how to test" instructions. How would everyone feel if I got rid of the about section and moved that contents to the "how to test" section? @line47 @pwolfert

Reply:
Yeah, that's a good idea.

Reply:
Ditto 👍🏼